signer/core/apitypes/signed_data_internal_test.go

```diff
@@ -244,45 +247,2470 @@ func TestConvertAddressDataToSlice(t *testing.T) {
 func TestTypedDataArrayValidate(t *testing.T) {
 	t.Parallel()
 
-	typedData := TypedData{
+	typedData0 := TypedData{
```

Comment:
Can I suggest that we move the typed data into files and parse them in the tests? The majority of this file looks like it's just TypedData definitions 😢

Reply:
Thanks @pano-skylakis, I've moved the test data to an external json file.